java/ql/integration-tests/java/query-suite/java-code-quality.qls.expected

```diff
@@ -1,4 +1,3 @@
-ql/java/ql/src/Language Abuse/TypeVariableHidesType.ql
```

Comment:
This query has `precision medium`. We've already discussed with @owen-mc that this can be removed from the quality suite.